Example/ios/Podfile.lock

```diff
@@ -435,9 +435,10 @@ PODS:
     - RCT-Folly (= 2021.07.22.00)
     - React-Core
     - ReactCommon/turbomodule/core
-  - RNScreens (3.31.0-rc.1):
+  - RNScreens (3.31.1):
```

Comment:
This is fine, however it would be nice to prepare separate PR with update of podfiles in all examples. Usually we try to not include changes in Podfile.lock & yarn.lock in regular PRs, but this is only a thumb rule.

Reply:
I'll prepare separate PR with bump

Reply:
And revert this change here.

Reply:
Bump in #2133